Gemfile.rails_version

```diff
@@ -2,13 +2,15 @@
 # separate fragment so that it can be sourced from the test
 # application's Gemfile in addition to the main development Gemfile.
 
-env_rails_version = ENV["RAILS_VERSION"] || '4.2.10'
+env_rails_version = ENV['RAILS_VERSION'] || '5.1.7'
 
 gem "rails", "~> #{env_rails_version}"
 
 case env_rails_version
 when "master"
   {github: "rails/rails"}
+when /^5.1/
+  gem 'bundler', '~> 2.1'
```

Comment:
I had to set this locally so it would bundle with the right Bundler- not sure if that is something we want to set here or in the gemspec